src/tools/tidy/src/issues.txt

```diff
@@ -183,8 +183,8 @@
 "ui/async-await/issue-67651.rs",
 "ui/async-await/issue-67765-async-diagnostic.rs",
 "ui/async-await/issue-68112.rs",
-"ui/async-await/issue-68523.rs",
 "ui/async-await/issue-68523-start.rs",
+"ui/async-await/issue-68523.rs",
```

Comment:
PR shouldn't affect ordering here, why it changed?

Reply:
preexisting: these weren't sorted, see https://github.com/rust-lang/rust/pull/122161#discussion_r1517107993

Reply:
@onur-ozkan Can you rebase this? It should remove these diffs since master should contain the relevant fix.